main/export-list.js

```diff
@@ -2,6 +2,7 @@
 'use strict';
 
 const {dialog} = require('electron');
+const {join: pathJoin} = require('path');
```

Comment:
There's no benefit of using destructuring here. `const path = require('path')` would be better.

Reply:
I agree 👍